mp/src/game/client/momentum/ui/HUD/hud_versioninfo.cpp

```diff
@@ -47,7 +47,13 @@ void CHudVersionInfo::VidInit()
 {
     KeyValuesAD loc("Version");
     loc->SetWString("verLabel", g_pVGuiLocalize->FindSafe("#MOM_StartupMsg_Alpha_Title"));
+
+#ifdef DEBUG
+    loc->SetString("verNum", "[Development Build]");
+#else
     loc->SetString("verNum", MOM_CURRENT_VERSION);
+#endif
```

Comment:
Yeah exactly what Brae said, and an easy way to do this would be:

```suggestion
    loc->SetString("verNum", 
#ifdef DEBUG
    "[Development Build] "
#endif
    MOM_CURRENT_VERSION);
```